arches/app/models/migrations/9437_include_map_source_and_map_layer_permissions.py

```diff
@@ -0,0 +1,15 @@
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('models', '9436_relational_view_null_geom_support'),
```

Comment:
It looks like somebody added a new migration, so this dependency needs to be updated to: `9055_add_branch_publication_to_node`

Reply:
dependency updated